fecgov/openFEC
# Fix MV refresh flow

## Summary (required)

- Resolves #3871
ofec_reports_house_senate_mv
ofec_reports_presidential_mv
ofec_reports_pac_party_mv

referencing ofec_filings_amendments_all_vw (the interface of ofec_filings_amendments_all_mv).  Updates manage.py and flow.py files so ofec_filings_amendments_all_mv will be refreshed BEFORE these MVs.  

## How to test the changes locally
 download the branch to local machine, run ```invoke create_sample_db```.  review the output of the MV refresh at the end of the process.  Make sure ofec_filings_amendments_all_mv is refreshed before the ofec_reports_house_senate_mv
ofec_reports_presidential_mv
ofec_reports_pac_party_mv

## Impacted areas of the application
List general components of the application that this PR will affect:

The API has direct impact from this change.  Data in the above reports MVs will be more correct after this change.

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()
